docs/user-guide/command-line-interface.md

```diff
@@ -450,3 +450,11 @@ ESLint supports `.eslintignore` files to exclude files from the linting process
     **/vendor/*.js
 
 A more detailed breakdown of supported patterns and directories ESLint ignores by default can be found in [Configuring ESLint](configuring.md#ignoring-files-and-directories).
+
+## Exit codes
+
+ESLint will exit with one of the following exit codes:
+
+* `0`: Linting was successful and there are no linting errors. If the `--max-warnings` flag is set to `n`, the number of linting warnings is at most `n`.
```

Comment:
I'm not sure this is valuable but I'm throwing it out anyway: Should we explicitly note that `--help` or `--version` returns exit code of 0?

Reply:
This is also the case for other flags like `--init` and `--print-config`. Maybe I could add a qualifier that these exit codes only apply when ESLint is used to lint files.